docs/services/azure-services-guide.md

```diff
@@ -0,0 +1,530 @@
+```
+
+* Define an Azure `Provider` and `Secret` in `azure-provider.yaml`:
+```yaml
+---
+# Azure Admin service account secret - used by Azure Provider
+apiVersion: v1
+kind: Secret
+metadata:
+  name: demo-provider-azure
+  namespace: azure
+type: Opaque
+data:
+  credentials: BASE64ENCODED_AZURE_PROVIDER_CREDS
+---
+# Azure Provider with service account secret reference - used to provision resources
+apiVersion: azure.crossplane.io/v1alpha2
+kind: Provider
+metadata:
+  name: demo-azure
+  namespace: azure
+spec:
+  credentialsSecretRef:
+    name: demo-provider-azure
+    key: credentials
+```
+
+#### Create Provider
+* Create Azure provider:
+```bash
+sed "s/BASE64ENCODED_AZURE_PROVIDER_CREDS/$BASE64ENCODED_AZURE_PROVIDER_CREDS/g" azure-provider.yaml | kubectl create -f -
+```
+
+* Verify Azure provider was successfully registered by the crossplane
+```bash
+kubectl get providers.azure.crossplane.io -n azure
+kubectl get secrets -n azure
+```
+
+#### Non-Portable Resource Classes
+Non-portable resource classes are used to define a reusable configuration for a specific managed service. Wordpress requires a MySQL database, which can be satisfied by an [Azure Database for MySQL](https://azure.microsoft.com/en-us/services/mysql/) instance.
+
+* Define an Azure MySQL `SQLServerClass` in `azure-mysql-class.yaml`:
+```yaml
+---
+apiVersion: database.azure.crossplane.io/v1alpha2
+kind: SQLServerClass
+metadata:
+  name: standard-azure-mysql
+  namespace: azure
+specTemplate:
+  adminLoginName: myadmin
+  resourceGroupName: RESOURCE_GROUP_NAME
+  location: EAST US
+  sslEnforced: false
+  version: "5.6"
+  pricingTier:
+    tier: GeneralPurpose
+    vcores: 2
+    family: Gen5
+  storageProfile:
+    storageGB: 25
+    backupRetentionDays: 7
+    geoRedundantBackup: false
+  providerRef:
+    name: demo-azure
+    namespace: azure
+  reclaimPolicy: Delete
+```
+
+* Create the `SQLServerClass`:
+```bash
+sed "s/RESOURCE_GROUP_NAME/$RESOURCE_GROUP_NAME/g" azure-mysql-class.yaml | kubectl apply -f -
+```
+
+* You should see the following output:
+```bash
+sqlserverclass.database.azure.crossplane.io/standard-azure-mysql created
+```
+
+* You can verify creation with the following command and output:
+
+*Command*
+```bash
+kubectl get sqlserverclasses -n azure
+```
+*Output*
+```bash
+NAME                   PROVIDER-REF   RECLAIM-POLICY   AGE
+standard-azure-mysql   demo-azure     Delete           11s
+```
+
+You are free to create more Azure `SQLServerClass` instances to define more potential configurations. For instance, you may create `large-azure-mysql` with field `storageGB: 100`.
+
+#### Namespaces
+Kubernetes namespaces allow for separation of environments within your cluster. You may choose to use namespaces to group resources by team, application, or any other logical distinction. For this demo, we will create a namespace called `app-project1-prod`, which we will use to group our Wordpress resources.
```

Comment:
app-project1-prod -> app-project1-dev to be consistent with https://github.com/crossplaneio/crossplane/blob/7fbcc6acdf3bd7c4bd1de4a45864e445e074dc14/docs/using-services.md#c-app-project-deploy-wordpress-with-a-managed-mysql-instance